_opensearch/rest-api/multi-search.md

```diff
@@ -59,6 +59,7 @@ All multi-search URL parameters are optional. Some can also be applied per-searc
 Parameter | Type | Description | Supported in metadata line
 :--- | :--- | :---
 allow_no_indices | Boolean | Whether to ignore wildcards that don't match any indices. Default is `true`. | Yes
+cancel_after_time_interval | Time | The time after which the search request will be cancelled. Supported at both parent and child request levels. The order of precedence is:<br> 1. Child-level parameter<br> 2. Parent-level parameter<br> 3. [Cluster setting](../cluster-settings).<br>Default is -1. | Yes
```

Comment:
Adjust the cluster settings link to the following format `{{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-settings`.

Reply:
Thank you! Changed links.